web/components/og/graph.tsx

```diff
@@ -29,27 +31,23 @@ export function Graph(props: {
   const stops = computeColorStops(data, color, px)
 
   return (
-    <svg width={w * (scaleX ?? 1)} height={h} viewBox={`0 0 ${w} ${h}`}>
-      <clipPath id={clipId}>
-        <rect x={0} y={0} width={w * (scaleX ?? 1)} height={h} />
-      </clipPath>
+    <svg width={w} height={h} viewBox={`0 0 ${w} ${h}`}>
```

Comment:
Have you tried these changes with the weekly summary og previews?

Reply:
not yet. where do I see weekly summaries?

Reply:
It's at the /update endpoint, you should be able to see that by seeing usages of the buildOgUrl

Reply:
left a comment on discord - I know where the code is, I just doin't know what an og image url for the weekly profit in the wild looks like because I don't know where to see my or other people's weekly profit